src/cleanvision/utils/utils.py

```diff
@@ -37,8 +37,10 @@ def get_filepaths(
     a list of image filenames sorted numerically and alphabetically
     """
 
-    if not os.path.isdir(dir_path):
-        raise NotADirectoryError
+    # this check means we cannot have dir_path be something like this: /256_ObjectCategories/*/
```

Comment:
can you write an appropriate check? The idea is to check if a valid path exists here

Reply:
not sure how to check this, because glob expressions are allowed to have no matches in the glob.glob function

I instead added a print warning if no images were found